src/App.tsx

```diff
@@ -1,24 +1,244 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable */ 
+import React, { useState, useEffect} from 'react';
 import { UserWarning } from './UserWarning';
+import classNames from 'classnames';
+import { getTodos, addNewTodo, deleteTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { Error } from './utils/Error';
 
-const USER_ID = 0;
+const USER_ID = 6752;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState('all');
+  const [errorMessage, setErrorMessage] = useState<Error>(Error.NoError);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [showNotification, setShowNotification] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loading, setLoading] = useState(false);
+
+  const loadTodos = async () => {
+    try {
+      setLoading(true);
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      setErrorMessage(Error.Loading);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
-  return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-add-and-delete#react-todo-app-add-and-delete">React Todo App - Add and Delete</a>
-      </p>
+  const getFilteringTodos = ():Todo[] => {
```

Comment:
```suggestion
  const getFilteredTodos = () => {
```
the function return type will be automatically detected - TS is smart enough for it :)